src/lint/latex-formatter/tex-fmt.ts

```diff
@@ -43,9 +43,13 @@ async function formatDocument(document: vscode.TextDocument, range?: vscode.Rang
         })
     })
 
-    process.stdin?.write(document.getText(range))
+    // write the document to the process, and add a newline at the end
```

Comment:
As this is a workaround of an external piece of software, please add corresponding documents describing the date, version of tex-fmt, and the context of the workaround.